packages/api-client/src/odsql/clauses.ts

```diff
@@ -1,11 +1,36 @@
 import { field, string } from '.';
 
+const formatFields = (fields: string[]) => {
+    if (!fields || fields.length === 0) return '';
+    return `${fields.map(field).join(', ')},`;
+};
+
+/**
+ * Perform full-text search on all fields. It matches the text fields that contain strings beginning with the searched string.
```

Comment:
Looks like the description for the `textStartWith` function.

Also apply for the `textSuggest` function